src/dotnet-interactive.Tests/StdIoKernelConnectorTests.cs

```diff
@@ -120,11 +120,38 @@ public async Task it_can_create_a_proxy_kernel_to_more_than_one_remote_subkernel
                 SupportedKernelCommands = fsharpKernelInfo.SupportedKernelCommands
             };
 
+            const string installHttpKernel =
+                """
+                #r "Microsoft.DotNet.Interactive.HttpRequest.dll"
+                using Microsoft.DotNet.Interactive;
+                using Microsoft.DotNet.Interactive.HttpRequest;
+                HttpRequestKernelExtension.Load(KernelInvocationContext.Current.HandlingKernel.RootKernel);
+                """;
+
+            result = await csharpProxyKernel.SubmitCodeAsync(installHttpKernel);
+            result.Events.Last().Should().BeOfType<CommandSucceeded>();
+
+            var httpKernelInfo = result.Events.OfType<KernelInfoProduced>().Should().ContainSingle().Which.KernelInfo;
+            using var httpProxyKernel = await connector.CreateProxyKernelAsync(remoteInfo: httpKernelInfo);
+            var expectedHttpKernelInfo = new KernelInfo("http")
+            {
+                IsProxy = true,
+                IsComposite = false,
+                LanguageName = httpKernelInfo.LanguageName,
+                LanguageVersion = httpKernelInfo.LanguageVersion,
+                RemoteUri = httpKernelInfo.Uri,
+                SupportedDirectives = httpKernelInfo.SupportedDirectives,
+                SupportedKernelCommands = httpKernelInfo.SupportedKernelCommands
+            };
+
             csharpProxyKernel.Name.Should().Be(csharpKernelInfo.LocalName);
             csharpProxyKernel.KernelInfo.Should().BeEquivalentTo(expectedCSharpKernelInfo);
 
             fsharpProxyKernel.Name.Should().Be("fsharp2");
             fsharpProxyKernel.KernelInfo.Should().BeEquivalentTo(expectedFSharpKernelInfo);
+
```

Comment:
When doing multiple assertions in a single test, it's best to add `using var _ = new AssertionScope();` beforehand. This will allow us to see all failures, not just the first.

Reply:
Ah it is already present above on line 95.